pkg/gofr/datasource/file/ftp/interface.go

```diff
@@ -0,0 +1,106 @@
+package ftp
+
+import (
+	"context"
+	"io"
+	"os"
+	"time"
+)
+
+// FileSystem : Any simulated or real filesystem should implement this interface.
```

Comment:
You should explain this method has this signature because you have to comply ftp.FileSystem